edgraph/server.go

```diff
@@ -1570,6 +1648,73 @@ func parseRequest(qc *queryContext) error {
 	return validateQuery(qc.dqlRes.Query)
 }
 
+// addQueryIfUnique adds dummy queries in the request for checking whether predicate is unique in the db
+func addQueryIfUnique(qc *queryContext) {
+	ctx := context.WithValue(context.Background(), schema.IsWrite, false)
+	if len(qc.gmuList) == 0 {
+		return
+	}
+
+	qc.uniqueVars = map[string]string{}
+	count := 1
```

Comment:
can we use the same thing as we are using for map keys and skip using the count variable?